### ISSUE TITLE
duplicate file is created if changed file is uploaded

### ISSUE THREAD

**Issue by mmp** [google/skicka#40]
```
% echo foo > foo
% skicka upload foo /foo
[...]
% echo bar >> foo
% skicka upload foo /foo
[...]
% skicka ls -l /foo
skicka: /foo: multiple files found
```

**Comment by pnu** [comment]
btw, same issue affects `upload`, when it's getting list of files to upload:
``` 
$ mkdir foodir
$ echo asdasd >foodir/foo
$ skicka upload foodir foodir
[…]
$ echo asdasd >>foodir/foo
$ skicka upload foodir foodir
[…]
$ echo asdasd >>foodir/foo
$ skicka upload foodir foodir
skicka: Getting list of files to upload... skicka: foodir/foo: duplicate file found on Google Drive

**Comment by mmp** [comment]
This should be fixed in ac702d253ea3389ec71c41a2f85f218b1397e8d5.

(There may still be cases where dupes are created when we don't recover from 500 errors properly, as suggested by issue #18, but this fixes the fully reproducible case for this case here.)
